man/os-release.xml

```diff
@@ -355,9 +355,12 @@
 
           <listitem><para>The date at which support for this version of the OS ends. (What exactly "lack of
           support" means varies between vendors, but generally users should assume that updates, including
-          security fixes, will not be provided.) The value is a date in the format
-          <literal>YYYY-MM-DD</literal>, and specifies the last day on which support <emphasis>is</emphasis>
-          provided.</para></listitem>
+          security fixes, will not be provided.) The value is a date in the ISO 8601 format
```

Comment:
i think this is slightly misleading. ISO 8601 allows so many other ways to specify dates and times. Here, we want to allow a small subset of it. (A subset that also happens to be compatible with our `systemd.time` syntax, btw.)

Hence I would clarify this, and write something like this:

```
The value is a date in the format <literal>YYYY-MM-DD</literal> (this date format is valid special case of both ISO 8601 and systemd.time(5)).
```

I think that makes clearer that we do not allow all ISO 8601 time formats here.